crates/workspace/src/pane_group.rs

```diff
@@ -902,6 +930,11 @@ mod element {
             self
         }
 
+        pub fn with_is_leaf_pane_mask(self, mask: Vec<bool>) -> Self {
```

Comment:
```suggestion
        fn with_is_leaf_pane_mask(self, mask: Vec<bool>) -> Self {
```

Since the PR changes a single module only, we do not seem to need to expose any extra public API?